frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/StaticEmbedSetupPane.tsx

```diff
@@ -0,0 +1,107 @@
+import { t } from "ttag";
+import { Tabs } from "metabase/ui";
+
+import type { ParametersSettingsProps } from "./ParametersSettings";
+import { ParametersSettings } from "./ParametersSettings";
+import type { AppearanceSettingsProps } from "./AppearanceSettings";
+import { AppearanceSettings } from "./AppearanceSettings";
+import type { OverviewSettingsProps } from "./OverviewSettings";
+import { OverviewSettings } from "./OverviewSettings";
+
+const TABS = {
+  Overview: "overview",
+  Parameters: "parameters",
+  Appearance: "appearance",
+};
+
+type StaticEmbedSetupPaneProps = OverviewSettingsProps &
```

Comment:
Do we have any other mechanisms for storing redundant information across components besides redux? I feel like this could be a good candidate for a provider or some sort of injection but I'm not totally sure in how that would affect the complexity/coupling here